### DISCUSSION TITLE
Missing indexed documents in Kibana

### DISCUSSION THREAD

**Discussion by lutacluny** [dadoonet/fscrawler#1590] [Q&A]
I followed the [steps](https://fscrawler.readthedocs.io/en/latest/installation.html#using-docker-compose) to use Kibanas Workplace Search. Only 3 of 5  documents appear in Kibana. One pdf with ~5000kb and one xlsx with ~100KB is missing. But the _status.json file mentions that 5 documents have been indexed.

{
  "name" : "idx",
  "lastrun" : "2023-01-24T08:46:54.982196673",
  "indexed" : 5,
  "deleted" : 0
}

Why do not all indexed documents appear in Workplace Search?

**Comment by dadoonet** [comment]
Could you run it again with the `--debug` option and share the logs here?
Also please use the `--restart` option so it will pick up again the documents...

**Reply by lutacluny** [reply]
10:38:30,081 [36mDEBUG[m [f.p.e.c.f.t.w.WPSearchBulkResponse] {63d25820e30e68ebd03bbf22={"results":[{"id":"a7a0f0e78df8a7616aeeff5c9265fd","errors":[]},{"id":"614064efd8827d09e1483a3f6ec7bbb","errors":[]},{"id":"ae562f32a4de6b4ff6c687834207979","errors":[]},{"id":"2e3c57e335edda7378a1db71c5133f2","errors":[]},{"id":"5492aa9effe5b5bc5fcccc91b7111e8","errors":[]},{"id":"ad41caac2321feec77d27f4fe832ca3","errors":["Exceeds maximum allowed document size of 102400 bytes"]},{"id":"daccc356396aa4a7515e84de536773a","errors":["Exceeds maximum allowed document size of 102400 bytes"]}]}} 

How can I increase the maximum allowed document size?

**Reply by lutacluny** [reply]
I solved it by adding `- workplace_search.content_source.document_size.limit=100000kb` into `docker-compose.yaml` at `elasticsearch.environmet`.

**Reply by dadoonet** [reply]
Amazing! @lutacluny  Would you like to contribute that tip to the documentation?

**Reply by lutacluny** [reply]
Yes, but `make html` does not work. ModuleNotFoundError: No module named 'ConfigParser' Do you know this issue?
